ocrd_zip.md

```diff
@@ -120,6 +120,18 @@ Wrong:
 All files except `mets.xml` itself that are contained in `data` directory must
 be referenced in a `mets:file/mets:Flocat` in the `mets.xml`.
 
+## Optional metadata about the payload
+
+In addition to the actual data files in `/data`, the following metadata files
+are allowed to be present in the root of the bag:
+
+* `README.md`: An extended, human-readable description of the dataset in the Markdown syntax
+* `Makefile`: A GNU make build file to reproduce the data in `/data`.
+* `build.sh`: A bash script to reproduce the data in `/data`.
```

Comment:
Ideally for reproducibility, provenance metadata should be used.

Reply:
Provenance metadata is not always available, such as for ground truth, test data, example sets etc.